src/crate/client/sqlalchemy/tests/dialect_test.py

```diff
@@ -59,31 +67,56 @@ class Character(self.base):
         self.character = Character
         self.session = Session()
 
-    def test_pks_are_retrieved_depending_on_version_set(self):
+    def test_pks_are_retrieved_depending_on_version_set_old(self):
+        """
+        Test the old pk retrieval.
+        """
         meta = self.character.metadata
-
-        # test the old pk retrieval
         insp = inspect(meta.bind)
         self.engine.dialect.server_version_info = (0, 54, 0)
-        fake_cursor.rowcount = 1
-        fake_cursor.fetchone = MagicMock(return_value=[["id", "id2", "id3"]])
+
+        # Setup fake cursor.
```

Comment:
```suggestion
```

See https://kevlinhenney.medium.com/comment-only-what-the-code-cannot-say-dfdb7b8595ac